frontend/src/container/AlertHistory/Timeline/GraphWrapper/GraphWrapper.tsx

```diff
@@ -29,20 +30,22 @@ function GraphWrapper({
 
 	const startTime = urlQuery.get(QueryParams.startTime);
 
-	const [isPlaceholder, setIsPlaceholder] = useState(false);
+	const [isVerticalGraph, setIsVerticalGraph] = useState(false);
 
 	useEffect(() => {
-		if (startTime) {
-			const startTimeDate = new Date(startTime);
-			const now = new Date();
-			const twentyFourHoursAgo = new Date(now.getTime() - 24 * 60 * 60 * 1000);
+		const checkVerticalGraph = (): void => {
+			if (startTime) {
+				const startTimeDate = dayjs(Number(startTime));
+				const twentyFourHoursAgo = dayjs().subtract(
+					HORIZONTAL_GRAPH_HOURS_THRESHOLD,
+					'hour',
```

Comment:
Please add this 'hour'  as a constant. 
Rest LGTM.

Reply:
Done